CONTRIBUTING.adoc

```diff
@@ -66,6 +66,8 @@ added after the original pull request but before a merge.
   than cosmetic changes).
 * Add some Javadocs.
 * A few unit tests would help a lot as well -- someone has to do it.
+* Verification tasks, including tests and stylecheck, can be executed
```

Comment:
Typo? Should be Checkstyle rather than stylecheck, I think.

Reply:
Ouch!  Thanks! :)